bokeh/tests/test_plotting.py

```diff
@@ -25,6 +25,16 @@ def test_figure(self):
         r = plt.figure()
         self.assertNotEqual(p, r)
         self.assertNotEqual(q, r)
+        p = plt.figure(width=100, height=120)
+        self.assertEqual(p.plot_width, 100)
+        self.assertEqual(p.plot_height, 120)
+        p = plt.figure(plot_width=100, plot_height=120)
+        self.assertEqual(p.plot_width, 100)
+        self.assertEqual(p.plot_height, 120)
+        p = plt.figure(plot_width=100, width=120)
+        self.assertIsNone(p)
+        p = plt.figure(plot_height=100, height=120)
+        self.assertIsNone(p)
```

Comment:
Yep, testing for the warning (or the exception if it is changed to an exception) would be good.